libs/source/Socket.cpp

```diff
@@ -79,17 +78,16 @@ void Socket::setLocalSocket(const char *ipaddr, int port) {
 
   int ret = inet_aton(ipaddr, &localSockAddr.sin_addr);
   if (ret == 0) {
```

Comment:
Isn't -1 also a return possibility here?

Reply:
nope: inet_aton returns 0 or 1 according to man page